client/state/selectors/get-past-billing-transactions.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import createSelector from 'lib/create-selector';
+import { getBillingTransactions } from './';
+
+/**
+ * Returns all past billing transactions.
+ * Returns null if the billing transactions have not been fetched yet.
+ *
+ * @param  {Object}  state   Global state tree
+ * @return {?Array}          An array of past transactions
+ */
+const getPastBillingTransactions = createSelector(
```

Comment:
We don't need to really need to memo here or in `getUpcomingBillingTransactions. We usually use it when we need to avoid work like when we manipulate structures eg filter/map/find.

This can be written as:
```javascript
export default function getPastBillingTransactions( state ) {
    const transactions = getBillingTransactions( state );
    return get( transactions, 'past', null );
} 
```